src/components/Video/index.tsx

```diff
@@ -21,10 +24,45 @@ const Video: React.FC<VideoProps> = (props) => {
     env,
     autoplay = false,
     playthrough = false,
-    // default 16:9
-    aspectRatio = 0.5625,
+    viewportPercentage = 75,
+    shrinkToFit = false,
   } = props;
   const muted = autoplay;
+  const containerRef = useRef();
+  const [aspectRatio, setAspectRatio] = useState(9 / 16); // default 16:9 (ratio is height / width)
```

Comment:
and then we'd need to take in aspectRatio in the initializer 
```suggestion
  const [aspectRatio, setAspectRatio] = useState(incomingAspectRatio); // default 16:9 (ratio is height / width)
```

Reply:
If I do this, the algorithm will set the initial aspect ratio, and overwrite it when the DOM is updated with a calculated value.  I kind of see the `incomingAspectRatio` as a hard override and this would not be so.